### ISSUE TITLE
libdagmc.so.0 not found (in Travis build)

### ISSUE THREAD

**Issue by asmeurer** [conda-forge/pyne-feedstock#22]
```pytb
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/home/travis/build/ergs/transmutagen/transmutagen/__init__.py", line 29, in <module>
    from .origen import (execute_origen, origen_to_array,
  File "/home/travis/build/ergs/transmutagen/transmutagen/origen.py", line 14, in <module>
    from pyne.utils import toggle_warnings
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/pyne/__init__.py", line 13, in <module>
    from .pyne_config import *
ImportError: libdagmc.so.0: cannot open shared object file: No such file or directory
```

Don't know if you're aware of this issue. It came up in this [Travis build](https://travis-ci.org/ergs/transmutagen/builds/347025413). I'm opening here because it looks like it's most likely an issue with the conda package. 

Here are the conda packages that were installed

```

    attrs:                         17.4.0-py_0                   conda-forge
    backports:                     1.0-py35_1                    conda-forge
    backports.functools_lru_cache: 1.5-py35_0                    conda-forge
    blas:                          1.1-openblas                  conda-forge
    boost-cpp:                     1.65.1-1                      conda-forge
    bzip2:                         1.0.6-1                       conda-forge
    ca-certificates:               2018.1.18-0                   conda-forge
    certifi:                       2018.1.18-py35_0              conda-forge
    cloog:                         0.18.0-0                      defaults   
    colorama:                      0.3.9-py35_0                  conda-forge
    cycler:                        0.10.0-py35_0                 conda-forge
    cython:                        0.27.3-py35_0                 conda-forge
    dbus:                          1.10.22-0                     conda-forge
    expat:                         2.2.5-0                       conda-forge
    fastcache:                     1.0.2-py35_0                  conda-forge
    fontconfig:                    2.12.6-0                      conda-forge
    freetype:                      2.8.1-0                       conda-forge
    gcc:                           4.8.5-7                       defaults   
    gettext:                       0.19.8.1-0                    conda-forge
    glib:                          2.55.0-0                      conda-forge
    gmp:                           6.1.2-0                       conda-forge
    gmpy2:                         2.0.8-py35_1                  conda-forge
    gst-plugins-base:              1.8.0-0                       conda-forge
    gstreamer:                     1.8.0-1                       conda-forge
    hdf5:                          1.10.1-2                      conda-forge
    icu:                           58.2-0                        conda-forge
    isl:                           0.12.2-0                      defaults   
    jinja2:                        2.10-py35_0                   conda-forge
    jpeg:                          9b-2                          conda-forge
    libffi:                        3.2.1-3                       conda-forge
    libgcc:                        7.2.0-h69d50b8_2              defaults   
    libgcc-ng:                     7.2.0-h7cc24e2_2              defaults   
    libgfortran:                   3.0.0-1                       defaults   
    libiconv:                      1.15-0                        conda-forge
    libpng:                        1.6.34-0                      conda-forge
    libstdcxx-ng:                  7.2.0-h7a57d05_2              defaults   
    libxcb:                        1.12-1                        conda-forge
    libxml2:                       2.9.7-0                       conda-forge
    lzo:                           2.10-0                        conda-forge
    markupsafe:                    1.0-py35_0                    conda-forge
    matplotlib:                    2.1.2-py35_0                  conda-forge
    metis:                         5.1.0-3                       conda-forge
    moab:                          5.0.2-3                       conda-forge
    mpc:                           1.0.3-4                       conda-forge
    mpfr:                          3.1.5-0                       conda-forge
    mpmath:                        1.0.0-py_0                    conda-forge
    ncurses:                       5.9-10                        conda-forge
    numexpr:                       2.6.4-py35_1                  conda-forge
    numpy:                         1.14.1-py35_blas_openblas_200 conda-forge [blas_openblas]
    openblas:                      0.2.20-7                      conda-forge
    openssl:                       1.0.2n-0                      conda-forge
    pcre:                          8.39-0                        conda-forge
    pip:                           9.0.1-py35_1                  conda-forge
    pluggy:                        0.6.0-py_0                    conda-forge
    py:                            1.5.2-py_0                    conda-forge
    pyflakes:                      1.6.0-py35_0                  conda-forge
    pyne:                          0.5.10-py35_blas_openblas_0   conda-forge [blas_openblas]
    pyparsing:                     2.2.0-py35_0                  conda-forge
    pyqt:                          5.6.0-py35_4                  conda-forge
    pytables:                      3.4.2-py35_7                  conda-forge
    pytest:                        3.4.1-py35_0                  conda-forge
    python:                        3.5.5-0                       conda-forge
    python-dateutil:               2.6.1-py35_0                  conda-forge
    pytz:                          2018.3-py_0                   conda-forge
    qt:                            5.6.2-7                       conda-forge
    readline:                      7.0-0                         conda-forge
    scikit-umfpack:                0.2.3-py35_blas_openblas_202  conda-forge [blas_openblas]
    scipy:                         1.0.0-py35_blas_openblas_201  conda-forge [blas_openblas]
    setuptools:                    38.5.1-py35_0                 conda-forge
    sip:                           4.18-py35_1                   conda-forge
    six:                           1.11.0-py35_1                 conda-forge
    sqlite:                        3.20.1-2                      conda-forge
    suitesparse:                   4.5.4-blas_openblas_201       conda-forge [blas_openblas]
    sympy:                         1.1.1-py35_0                  conda-forge
    tbb:                           2018_20171205-0               conda-forge
    tk:                            8.6.7-0                       conda-forge
    tornado:                       4.5.3-py35_0                  conda-forge
    wheel:                         0.30.0-py35_2                 conda-forge
    xorg-libxau:                   1.0.8-3                       conda-forge
    xorg-libxdmcp:                 1.1.2-3                       conda-forge
    xz:                            5.2.3-0                       conda-forge
    zlib:                          1.2.11-0                      conda-forge
```

**Comment by isuruf** [comment]
Looks like moab 4.* builds and 5.* builds are incompatible because of the change in https://github.com/conda-forge/moab-feedstock/pull/8/files#diff-44a73bcc045c193c3bd45da87994b03bL9

**Comment by scopatz** [comment]
Yeah, I think the solution for now is to pin pyne to MOAB v4.  I'll put in a PR

**Comment by scopatz** [comment]
Damn, pyne is running into build problems with new downstream data releases.  I recommend that you manually downgrade moab for the time being.

**Comment by scopatz** [comment]
Pyne needs to have a micro release that fixes these.

**Comment by gonuke** [comment]
This is being resolved in a series of PRs beginning with #34 that removes the dagmc dependency and then others that will carefully add it back in

**Comment by gonuke** [comment]
DAGMC dependency is now optional and not yet supported in conda-forge.  We are working on a robust conda-forge implementation of DAGMC after which we can add back the dagmc dependency here.
